### PR TITLE
Add CZI Viz JD

### PR DESCRIPTION
**Pull request by canyon289**

### CODE REVIEW

**Review by ahartikainen — APPROVED** [review]

File: CZI/viz_evaluation.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# ArviZ Visual Refresh, Python API Assessment, and other advice
+ArviZ is a Python and Julia libary for exploratory analysis of Bayesian models,
+and includes many [vizualizations](https://arviz-devs.github.io/arviz/examples/index.html)
```
Comment (OriolAbril) [inline CZI/viz_evaluation.md]:
```suggestion
and includes many [vizualizations](https://python.arviz.org/en/latest/examples/index.html)
```

File: CZI/viz_evaluation.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+The scope of this work is to assess update visual style of the package (all components including docs)
+and perform an assessment of the API (primary python), and offer any other advice 
+
+## Job description
+We are expecting a report covering the following points:
+
+* Analysis of available visualizations across Python and Julia
+
+  Evaluate and criticize current plots for visual style and effectiveness.
+  Critique the visual clarity of each
```
Comment (OriolAbril) [inline CZI/viz_evaluation.md]: I would remove this as I think this overlaps with the GSoD report goals.

File: CZI/viz_evaluation.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+If a question requires privacy you can also use the email below.
+
+## Applying for the job
+Applicants who wish to be hired to write the report should send an email to `arviz.devs@gmail.com`
+with the following information:
+
+* CV or Resume
+* Sample report
+
+  We ask you to choose one of [our plots](https://arviz-devs.github.io/arviz/api/plots.html) and write a sample report for it
```
Comment (OriolAbril) [inline CZI/viz_evaluation.md]:
```suggestion
  We ask you to choose one of [our plots](https://python.arviz.org/en/latest/api/plots.html) and write a sample report for it
```

**Review by OriolAbril — COMMENTED** [review]
I added a couple specific comments but I think we should first make sure we are on the same page in terms of goals as well as general roadmap and context.

### General context
We got some funding from CZI Round 4 last year which was mostly focused on julia development, improvement of educational materials and maintenance of the python library. The funds to pay for this job would come from here. 

We discussed how to distribute the money allocated for python maintenance with @agustinaarroyuelo and @aloctavodia mostly as of recently, but the discussion started on a general meeting and we have also had an open google doc for async discussion for a few months. Our current approach is to use part of this years money to design how the plotting module should look like so that we can then pay someone with next years money and money from CZI Round 5 to refactor the plotting module and implement this design.

I started [his wiki page](https://github.com/arviz-devs/arviz/wiki/ArviZ-1.0-ideas#plotting-modularity) with some ideas on this in early February as part of the discussions. It focuses on backend stuff because it is where I had more concrete ideas but I think all the user api needs to change too so that we can support much better facetting for users but also for us to avoid having to duplicate the facetting code in every single plot.

### Work plan

I was planning on dedicating time for this during the summer, with the goal of writing down a proof of concept with a couple facetting alternatives and a couple plots so that we can see what works and what doesn't and have a discussion on this (if we get round 5 funding we can also dedicate part of the funding early next year to testing and improving the design before implementing it). 

On this line, I think the best approach would be onboarding someone that can help with that process. That is, focusing on the design, implementing a (or a couple) proof of concept design(s) and testing them. Which would translate for example to critique of the current api and its limitations but no changes at all on the current codebase.

File: CZI/viz_evaluation.md (hunk 2 above)
Comment (canyon289) [reply CZI/viz_evaluation.md]: Thats fine if it overlaps, one person can move the baton forward and whenever a GSOD person comes they can provide their critique too. That way we get a distribution of opinions

**Comment by canyon289** [comment]
> On this line, I think the best approach would be onboarding someone that can help with that process. That is, focusing on the design, implementing a (or a couple) proof of concept design(s) and testing them. Which would translate for example to critique of the current api and its limitations but no changes at all on the current codebase.

We can do this too, lets write up another JD and find someone suitable for a goal. None of this work is mutually exclusive between GSOD and all CZI. Theyre all self reinforcing!

> Our current approach is to use part of this years money to design how the plotting module should look like so that we can then pay someone with next years money and money from CZI Round 5 to refactor the plotting module and implement this design

That sounds good! Do we not have enough funds to do both? Last I checked we have a lot of funds, much much more than the proposal here

File: CZI/viz_evaluation.md [hunk 4]
```diff
@@ -1,34 +1,39 @@
...
-* Analysis of available visualizations across Python and Julia
 
-  Evaluate and criticize current plots for visual style and effectiveness.
-  Critique the visual clarity of each
+1. Provide a design guide
 
-* Provide a design guide
-  This ideally would be in the form of an MPL, Bokeh, or other style template.
-  A README.md style guide should provided if deemed necessary as well.
+   This ideally would be in the form of a README.md style guide with explanation and references,
```
Comment (canyon289) [inline CZI/viz_evaluation.md:34]: Thanks Oriol!

File: CZI/viz_evaluation.md [hunk 5]
```diff
@@ -1,34 +1,39 @@
...
+   This role will require you to get hands on with code.
+   During this process documenting the improvements that could be made to the API and
+   code workflow itself is expected, updates to the code are not a priority.
+
+1. Analysis of available visualizations across Python and Julia
+
+   Evaluate and criticize current plots for visual style and effectiveness.
+   Critique the visual clarity of each
+
+1. Any other advice or tips
```
Comment (canyon289) [inline CZI/viz_evaluation.md]: * These are all marked one, are they the same priority or were they mean to be different

Comment (canyon289) [reply CZI/viz_evaluation.md]: I see the note above for descending priority

**Pull request merged**